### PR TITLE
DOC: Consistent naming for 'Insight Toolkit (ITK)'

### PR DESCRIPTION
**Pull request by thewtex**
As discussed with 199c38ed11da9fad4608d064767beaf0fd70efc8
925eb8fe771ae5edde2d2f4341c317dc11a9b96a
use the simpler and more widely known name 'Insight Toolkit (ITK)'.

### CODE REVIEW

File: Documentation/Doxygen/MainPage.dox [hunk 1]
```diff
@@ -6,15 +6,14 @@
  *
  * \section intro Introduction
  *
- * Welcome to the National Library of Medicine Insight Segmentation and
- * Registration Toolkit (ITK). ITK is an open-source sofware system to 
+ * Welcome to the Insight Toolkit (ITK). ITK is an open-source sofware system to
```
Comment (blowekamp) [inline Documentation/Doxygen/MainPage.dox]: I agree with changing the name, however this paragraph no longer has much of a descriptions of what ITK is. Consider changing to:
> Welcome to the Insight Toolkit (ITK). ITK is an open-source software system for segmentation and registration initially developed to support the Visible Human Project.

**Comment by blowekamp** [comment]
The consistency is good. But some places may need a brief description of with ITK something like:

The "Insight Toolkit" for segmentation and registration.
The "Insight Toolkit" for advanced bio-medical image analysis.

Both are lacking the full breadth of ITK, but any condensed summary will be.

**Comment by thewtex** [comment]
Good ideas -- I'll merge and follow-up with a more detailed description in another patch.

**Comment by thewtex** [comment]
Extended description added in https://github.com/InsightSoftwareConsortium/ITK/pull/1271

**Pull request merged**